cmd/orb-server/startcmd/params.go

```diff
@@ -629,6 +649,7 @@ func createFlags(startCmd *cobra.Command) {
 	startCmd.Flags().StringP(httpSignaturesEnabledFlagName, httpSignaturesEnabledShorthand, "", httpSignaturesEnabledUsage)
 	startCmd.Flags().StringP(casTypeFlagName, casTypeFlagShorthand, "", casTypeFlagUsage)
 	startCmd.Flags().StringP(ipfsURLFlagName, ipfsURLFlagShorthand, "", ipfsURLFlagUsage)
+	startCmd.Flags().String(cidVersionFlagName, "0", cidVersionFlagUsage)
```

Comment:
Does this mean to default to "0" if not provided?

Reply:
Oops, yes you're correct. That should be "1". I've fixed it now.

Reply:
Also, your if statement above is handling case "". Do you need to adjust this as well? cidVersionString != "1" && cidVersionString != ""